src/ollie.js

```diff
@@ -19,10 +19,12 @@ export default class Ollie {
       this.printWelcome();
       const answeringAnswers = await inquirer.prompt(this.openingQuestions());
       const surveyAnswers = await this.startSurvey(answeringAnswers);
-      const { localPath } = surveyAnswers;
-      this.printFinish(localPath);
+      const { localPath, remoteRepositoryUrl } = surveyAnswers;
+      const [, hostName, teamName, repoName] = remoteRepositoryUrl.match(/git@(\S+):(\S+)\/(\S+)\.git/);
```

Comment:
`[ , ]` Is this a forgotten comma?

Reply:
Nope. Array destructuring: ignore first element

Reply:
Looked over the fact that it was a regex, ok then